internal/install/static_snapshot_yml.go

```diff
@@ -69,4 +69,31 @@ services:
     depends_on:
       package-registry:
         condition: service_healthy
+
+  elastic-agent:
+    image: docker.elastic.co/beats/elastic-agent:${STACK_VERSION}
+    depends_on:
+      elasticsearch:
+        condition: service_healthy
+      kibana:
+        condition: service_healthy
+    healthcheck:
+      test: ["CMD", "grep", "Agent is starting", "/usr/share/elastic-agent/elastic-agent.log"]
+      retries: 30
+      interval: 1s
+    environment:
+    - "FLEET_ENROLL=1"
+    - "FLEET_ENROLL_INSECURE=1"
+    - "FLEET_SETUP=1"
+    - "KIBANA_HOST=http://kibana:5601"
+    volumes:
```

Comment:
I did not, but will do.

Reply:
Hmmm, interesting. It seems to work for `package-registry` and `elasticsearch` but not for `kibana`:

### Starting up only `package-registry` ✔️ 

```
$ elastic-package stack up --version 8.0.0-SNAPSHOT -d -v -s package-registry
2020/09/01 10:34:28 DEBUG Enable verbose logging
Boot up the Elastic stack
2020/09/01 10:34:28 DEBUG running command: /usr/local/bin/docker-compose -f /Users/shaunak/.elastic-package/stack/snapshot.yml -p elastic-package-stack build package-registry package-registry_is_ready
2020/09/01 10:34:53 DEBUG running command: /usr/local/bin/docker-compose -f /Users/shaunak/.elastic-package/stack/snapshot.yml -p elastic-package-stack down
2020/09/01 10:34:54 DEBUG running command: /usr/local/bin/docker-compose -f /Users/shaunak/.elastic-package/stack/snapshot.yml -p elastic-package-stack up -d package-registry package-registry_is_ready
Done
```
```
$ docker ps
CONTAINER ID        IMAGE                                    COMMAND                  CREATED             STATUS                    PORTS                      NAMES
2ff3fbee2ecb        elastic-package-stack_package-registry   "./package-registry …"   18 seconds ago      Up 17 seconds (healthy)   127.0.0.1:8080->8080/tcp   elastic-package-stack_package-registry_1
```

### Starting up only `elasticsearch` ✔️ 

```
$ elastic-package stack up --version 8.0.0-SNAPSHOT -d -v -s elasticsearch
2020/09/01 10:41:34 DEBUG Enable verbose logging
Boot up the Elastic stack
2020/09/01 10:41:34 DEBUG running command: /usr/local/bin/docker-compose -f /Users/shaunak/.elastic-package/stack/snapshot.yml -p elastic-package-stack build elasticsearch elasticsearch_is_ready
2020/09/01 10:41:34 DEBUG running command: /usr/local/bin/docker-compose -f /Users/shaunak/.elastic-package/stack/snapshot.yml -p elastic-package-stack down
2020/09/01 10:41:35 DEBUG running command: /usr/local/bin/docker-compose -f /Users/shaunak/.elastic-package/stack/snapshot.yml -p elastic-package-stack up -d elasticsearch elasticsearch_is_ready
Done
```
```
$ docker ps
CONTAINER ID        IMAGE                                                          COMMAND                  CREATED              STATUS                        PORTS                                NAMES
826522aaf207        docker.elastic.co/elasticsearch/elasticsearch:8.0.0-SNAPSHOT   "/tini -- /usr/local…"   About a minute ago   Up About a minute (healthy)   127.0.0.1:9200->9200/tcp, 9300/tcp   elastic-package-stack_elasticsearch_1
```

### Starting up only `kibana` ❌ 

```
$ elastic-package stack up --version 8.0.0-SNAPSHOT -d -v -s kibana
2020/09/01 10:36:42 DEBUG Enable verbose logging
Boot up the Elastic stack
2020/09/01 10:36:42 DEBUG running command: /usr/local/bin/docker-compose -f /Users/shaunak/.elastic-package/stack/snapshot.yml -p elastic-package-stack build
2020/09/01 10:37:06 DEBUG running command: /usr/local/bin/docker-compose -f /Users/shaunak/.elastic-package/stack/snapshot.yml -p elastic-package-stack down
2020/09/01 10:37:07 DEBUG running command: /usr/local/bin/docker-compose -f /Users/shaunak/.elastic-package/stack/snapshot.yml -p elastic-package-stack up -d
Done
```
```
$ docker ps
CONTAINER ID        IMAGE                                                          COMMAND                  CREATED              STATUS                        PORTS                                NAMES
f00fbb57d861        docker.elastic.co/beats/elastic-agent:8.0.0-SNAPSHOT           "/usr/local/bin/dock…"   40 seconds ago       Up 39 seconds (healthy)                                            elastic-package-stack_elastic-agent_1
a3b35e4a30ad        docker.elastic.co/kibana/kibana:8.0.0-SNAPSHOT                 "/usr/local/bin/dumb…"   About a minute ago   Up About a minute (healthy)   127.0.0.1:5601->5601/tcp             elastic-package-stack_kibana_1
6f35f3833863        docker.elastic.co/elasticsearch/elasticsearch:8.0.0-SNAPSHOT   "/tini -- /usr/local…"   2 minutes ago        Up 2 minutes (healthy)        127.0.0.1:9200->9200/tcp, 9300/tcp   elastic-package-stack_elasticsearch_1
0c4cd0260cf1        elastic-package-stack_package-registry                         "./package-registry …"   2 minutes ago        Up 2 minutes (healthy)        127.0.0.1:8080->8080/tcp             elastic-package-stack_package-registry_1
```

Wonder why that is? 🤔 Investigating...

Reply:
Oh, it's because of this check: https://github.com/elastic/elastic-package/blob/ba7825e1b010cab16705f47c5c6d1fad332edd52/internal/stack/boot.go#L186-L188

I will update that to check for `elastic-agent` now as that is the "root" service in the dependency tree now.

Reply:
Updated in 730738123a978601e4e871e2958d16cfd5f8d18a.